std/polynomial/polynomial_test.go

```diff
@@ -31,82 +29,26 @@ func testEvalPoly(t *testing.T, p []int64, at int64, evaluation int64) {
 	assert := test.NewAssert(t)
 
 	witness := evalPolyCircuit{
-		P:          Polynomial(int64SliceToVariableSlice(p)),
+		P:          polynomial.Polynomial(int64SliceToVariableSlice(p)),
 		At:         at,
 		Evaluation: evaluation,
 	}
 
-	assert.CheckCircuit(&evalPolyCircuit{P: make(Polynomial, len(p))}, test.WithValidAssignment(&witness))
+	assert.CheckCircuit(&evalPolyCircuit{P: make(polynomial.Polynomial, len(p))}, test.WithValidAssignment(&witness))
 }
 
 func TestEvalPoly(t *testing.T) {
 	testEvalPoly(t, []int64{1, 2, 3, 4}, 5, 586)
 }
 
```

Comment:
WHy the tests removed?

Reply:
I had to change the name of the package to break an import cycle caused by the new hash registry.
The removed tests belong to private functions already covered by tests for their "parent"s.